### PR TITLE
User can search for sprays

### PR DESCRIPTION
**Pull request by jooliecodes**
https://github.com/julieluangpraxay/valorant-collections/assets/130415142/d0479907-55e0-4586-b18c-bff1caafbc40

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+  width: 65%;
+  margin: 0.8rem 0;
+  height: 20%;
+}
+
+.mobile-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  transition: 1s;
```
Comment (shawnkost) [inline css/styles.css:31]: This shouldn't be needed anymore

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+}
+
+.carousel {
+  margin-top: 1rem;
+  height: 200px;
+}
+
+.hidden {
+  display: none;
+}
```
Comment (shawnkost) [inline css/styles.css]: I would add `!important` to this, just to make sure that nothing overrides this class. If an element gets this class, we want to make sure it's always hidden regardless of what other styles it has

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+
+.sidenav .nav-button {
+  text-align: center;
+  padding: 3px 3px 3px 5px;
+  margin: 2.5rem 1.5rem;
+  text-decoration: none;
+  font-size: 25px;
+  color: #0e4071;
+  display: block;
+  transition: 0.3s;
```
Comment (shawnkost) [inline css/styles.css]: I don't think this is needed anymore

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+  box-shadow: 10px 10px 0 -2px #fdeaac, 10px 10px 0 0 black;
+  border-radius: 2rem;
+  background-color: #fdeaac;
+  border: 3px solid black;
+}
+
+.sidenav a:active {
+  transform: translateY(0.25rem);
+}
```
Comment (shawnkost) [inline css/styles.css:188]: Don't think this is needed anymore

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+  }
+
+  .search-wrapper {
+    margin-bottom: 3rem;
+  }
+}
+
+cursor:hover {
+  cursor: pointer;
+}
```
Comment (shawnkost) [inline css/styles.css:331]: This should be above your media queries

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+        const $cardWrapper = document.createElement('div');
+        const $iconImage = document.createElement('img');
+        const $textWrapper = document.createElement('div');
+        const $sprayTitle = document.createElement('p');
+
+        if (xhr.response.data[i].animationGif !== null) {
+          $iconImage.src = xhr.response.data[i].animationGif;
+        } else {
+          $iconImage.src = xhr.response.data[i].fullTransparentIcon;
+        }
```
Comment (shawnkost) [inline js/main.js]: Also needs an `alt` attribute

File: js/main.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+      const $textWrapper = document.createElement('div');
+      const $sprayTitle = document.createElement('p');
+
+      $sprayTitle.textContent = data.sprays[i].displayName;
+      if (data.sprays[i].animationGif !==
+         null) {
+        $iconImage.src = data.sprays[i].animationGif;
+      } else {
+        $iconImage.src = data.sprays[i].fullTransparentIcon;
+      }
```
Comment (shawnkost) [inline js/main.js]: Also needs an `alt` attribute

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Julie! Some changes requested here

File: css/styles.css (hunk 4 above)
Comment (jooliecodes) [reply css/styles.css:188]: This actually is for when the button inside the nav is clicked

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**